### ISSUE TITLE
Inline scripts and styles

### ISSUE THREAD

**Issue by szepeviktor** [johnbillion/query-monitor#110]
Could you mark localized scripts (`data`) and inline styles (`extra`) in Scripts and Styles?

**Comment by johnbillion** [comment]
Yeah I probably could. Thanks for the suggestion.

**Comment by crstauf** [comment]
I've been working on implementing this, and I'm getting close:

![screen shot 2016-07-22 at 2 44 07 pm](https://cloud.githubusercontent.com/assets/4573033/17067694/deb5a384-501a-11e6-8841-54ea32f4295e.png)

but can't determine the best way to handle large amounts of inline data:

![screen shot 2016-07-22 at 2 45 27 pm](https://cloud.githubusercontent.com/assets/4573033/17067719/f9668720-501a-11e6-864f-6e125d14e8e5.png)

@johnbillion any thoughts?

**Comment by szepeviktor** [comment]
Maybe get the first line and truncate is at 80 or 100 `ms_strlen()`.

**Comment by crstauf** [comment]
I'm wondering, if it's even worth it to show the inline data... it's in the source of the page, so perhaps just indicating that there is inline data is sufficient (as if you're interested, you could go look find it in the source)?

**Comment by crstauf** [comment]
hmmm yeah I think providing the first 60 characters or so would be enough; user could grab the characters and search for them in the source. will do that.

**Comment by szepeviktor** [comment]
> I'm wondering, if it's even worth it to show the inline data...

My issue was: "Could you mark ..."

**Comment by crstauf** [comment]
yeah, that's what got me thinking about even showing it... I do and don't like showing it; seems redundant, but also helpful for a quick glance. *thinking it over*

**Comment by crstauf** [comment]
thoughts, everyone?

![screen shot 2016-07-22 at 3 08 59 pm](https://cloud.githubusercontent.com/assets/4573033/17068328/4a542c66-501e-11e6-8923-f74a4cb97dca.png)

**Comment by szepeviktor** [comment]
Could you add the count? `data` and `2×after`

**Comment by crstauf** [comment]
@szepeviktor count how many items are in the `data` and `after` keys? `data` is only a string, but `after` can have multiple items... is that what you're wanting? a count of how many inline scripts have been added?

**Comment by crstauf** [comment]
@szepeviktor how's this?

![screen shot 2016-07-22 at 4 06 40 pm](https://cloud.githubusercontent.com/assets/4573033/17069765/5199e86e-5026-11e6-89c0-69b31ed93be8.png)

**Comment by szepeviktor** [comment]
OK.

**Comment by crstauf** [comment]
@johnbillion Is there something in particular you're wanting to do with this, that kept it from going through in 2016? Please let me know; happy to do what needs to be done to get this into QM core.

**Comment by johnbillion** [comment]
I think I decided that I wanted to expose more information than this, or in a better way, but I ended up having decision paralysis. I'll check out your updated PR, thanks.

**Comment by szepeviktor** [comment]
> decision paralysis

I know what that is!
